internal/cmd/app/generate_all.go

```diff
@@ -15,8 +15,11 @@
 package app
 
 import (
-	"github.com/AppsFlyer/terra-crust/internal/cmd/types"
 	"github.com/spf13/cobra"
+
+	"github.com/AppsFlyer/terra-crust/internal/cmd/types"
+	template_reader "github.com/AppsFlyer/terra-crust/internal/services/drivers/template-reader"
```

Comment:
These import aliases is a smell for faulty package naming (don't use `-` in folder name)